python/kvikio/pyproject.toml

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2021-2022, NVIDIA CORPORATION. All rights reserved.
+# Copyright (c) 2024, NVIDIA CORPORATION. All rights reserved.
```

Comment:
*nit*: I guess the automated copyright checker thinks this is a new file with no history? But I think this should be `2021-2024`?